File: MyLittlePony/Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyLittlePony
{
    class Card
    {
        private Player _player;
        private string _name;
        private string _id;
        private List<Property> _properties = new List<Property>();

        public Card(string name, string id, List<Property> property)
        {
            this.setName(name);
            this.setID(id);
            this._properties = property;
        }

        public Player getPlayer()
        {
            return this._player;
        }

        public void setPlayer(Player player)
        {
            this._player = player;
        }

        public string getName()
        {
            return this._name;
        }

        public void setName(string name)
        {
            this._name = name;
        }

        public string getID()
        {
            return this._id;
        }

        public void setID(string ID)
        {
            this._id = ID;
        }

        public List<Property> getProperties()
        {
            return _properties;
        }
    }
}

File: Classes/Domain/Model/Card.php
<?php
declare(strict_types=1);

namespace robertontl\mylittlepony\Classes\Domain\Model;

class Card
{
    /**
     * @var string
     */
    protected $id = '';

    /**
     * @var string
     */
    protected $name = '';

    /**
     * @var array
     */
    protected $properties = array();

    /**
     * @var int
     */
    protected $playerId;

    /**
     * @var string
     */
    protected $image = '';

    /**
     * @var bool
     */
    protected $trumped;

    public function setId(string $id): void
    {
        $this->id = $id;
    }

    public function getId(): string
    {
        return $this->id;
    }

    public function setName(string $name): void
    {
        $this->name = $name;
    }

    public function getName(): string
    {
        return $this->name;
    }

    public function setProperties(array $properties): void
    {
        $this->properties = $properties;
    }

    public function getProperties(): array
    {
        return $this->properties;
    }

    public function setPlayerId(int $playerId): void
    {
        $this->playerId = $playerId;
    }

    public function getPlayerId(): int
    {
        return $this->playerId;
    }

    public function setImage(string $image): void
    {
        $this->image = $image;
    }

    public function getImage(): string
    {
        return $this->image;
    }

    public function setTrumped(bool $trumped): void
    {
        $this->trumped = $trumped;
    }

    public function getTrumped(): bool
    {
        return $this->trumped;
    }
}
